client/src/components/CreatePost/Form/FirstSection.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Input } from "antd";
+import styled from "styled-components";
+import Head from "./Head";
+import { Section } from "../StyledModal";
+
+const TitleInput = styled(Input)``;
+const { TextArea } = Input;
+const TextInput = styled(TextArea)`
+  border-top: none;
+`;
+
+const First = ({
+  onChangeTitle,
+  onChangeDescription,
+  renderError,
+  formData,
+}) => {
+  return (
+    <Section>
+      <Head number={1} title="Write your post here" />
```

Comment:
Are you sure about the icons in Figma? The numbers are hardcoded to the image, it would be a different svg icon to each section.
About the second point, I don't  think this a header, I might give it a better name. You think is good to use the Heading? I would have to create new h elements in the Heading component.

Reply:
Is this line responsible for the bubble icons with number? If so we can grab the whole icon from Figma directly. Double click on the icon and right click to save as SVG. 

But I think we can skip this step for now since the current implementation looks fine. Just wanted to bring up that some icons can be accessed on Figma directly, that's all :)

![Screen Shot 2020-05-24 at 8 26 31 AM](https://user-images.githubusercontent.com/42228386/82757936-4b2c3680-9d98-11ea-9abf-077884ef764f.png)

Regarding the heading, you can use the component that @snowiesuet created related to #295. It is a Heading component that we want to resuse to stay consistent across the code base. Does that make sense?